### ISSUE TITLE
Resolve chained references to tokens

### ISSUE THREAD

**Issue by jorisw** [atanasster/babel-plugin-theme-ui#11]
I'm trying to make the following work:

```ts
// theme.ts
{
  colors: {
    text: 'content.primary',
    content: {
      primary: 'blues.1000',
    },
    blues: {
      '1000': '#0000ff',
...
```

However when compiled, the text is given style:

```css
body {
  color: blues.1000; /* Invalid value */
}
```

I was hoping it would resolve the two-step reference of `content.primary`, via `blues.1000`, all the way to `#0000ff`.

When I assign `blues.1000` directly to `colors.text`, the color token is correctly applied:

```css
body {
  color: #0000ff; /* Correctly resolved value */
}
```

**Comment by atanasster** [comment]
Thanks for report, just released 0.8.0 with support for double indirection as your example

Here is your example as a test for the update : https://github.com/atanasster/babel-plugin-theme-ui/blob/master/tests/fixtures/double-indirection/code.ts

**Comment by jorisw** [comment]
Indeed it works, many thanks!
